gramps/plugins/docgen/svgdrawdoc.py

```diff
@@ -128,7 +128,7 @@ def rotate_text(self, style, text, x, y, angle, mark=None):
         self.buffer.write(
             '<text ' +
             'x="%4.2f" y="%4.2f" ' % (xpos, ypos) +
-            'transform="rotate(%d %4.2f %4.2f)" ' % (angle, centerx, centery) +
+            'transform="rotate(%4.2 %4.2f %4.2f)" ' % (angle, centerx, centery) +
```

Comment:
I think you mean:
```suggestion
            'transform="rotate(%4.2f %4.2f %4.2f)" ' % (angle, centerx, centery) +
```

Reply:
Yes I did, thanks for checking! I amended the commit with the extra `f`.